freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -45,9 +48,13 @@ def __init__(self, pyxis=None):
             self._pyxis = pyxis
         else:
             if not conf.pyxis_server_url:
-                raise ValueError("'pyxis_server_url' parameter should be set")
+                raise ValueError("'PYXIS_SERVER_URL' parameter should be set")
             self._pyxis = Pyxis(conf.pyxis_server_url)
 
+        if not conf.freshmaker_root_url or "://" not in conf.freshmaker_root_url:
+            raise ValueError("'FRESHMAKER_ROOT_URL' parameter should be set to"
+                             "valid URL")
```

Comment:
Missing a space but might as well add the missing "a" as well.
```suggestion
                             "a valid URL")
```